addons/sourcemod/translations/zombieriot.phrases.zombienames.txt

```diff
@@ -4,647 +4,777 @@
 	{
 		"en" 		"This text isnt supposed to apear! Please report this to the admins!"
 		"ru" 		"Этот текст не должен появляться! Пожалуйста, сообщите об этом администраторам!"
+		"pt"		"Esta linha de texto não deveria estar aqui, seria uma boa ideia mostrar isso para os administradores."
 	}
 	"Headcrab Zombie"
 	{
 		"en" 		"Headcrab Zombie"
 		"ru" 		"Зомби"
+		"pt"		"Zumbi de Headcrab"
 	}
 	"Fortified Headcrab Zombie"
 	{
 		"en" 		"Fortified Headcrab Zombie"
 		"ru" 		"Крепкий зомби"
+		"pt"		"Zumbi de Headcrab Fortificado"
 	}
 	"Fast Zombie"
 	{
 		"en" 		"Fast Zombie"
 		"ru" 		"Быстрый зомби"
+		"pt"		"Zumbi Rápido"
 	}
 	"Fortified Fast Zombie"
 	{
 		"en" 		"Fortified Fast Zombie"
 		"ru" 		"Крепкий быстрый зомби"
+		"pt"		"Zumbi Rápido Fortificado"
 	}
 	"Torsoless Headcrab Zombie"
 	{
 		"en" 		"Torsoless Headcrab Zombie"
 		"ru" 		"Зомби без торса"
+		"pt"		"Zumbi de Headcrab sem pernas"
 	}
 	"Poison Zombie"
 	{
 		"en" 		"Poison Zombie"
 		"ru" 		"Ядовитый зомби"
+		"pt"		"Zumbi Venenoso"
 	}
 	"Fortified Poison Zombie"
 	{
 		"en" 		"Fortified Poison Zombie"
 		"ru" 		"Крепкий ядовитый зомби"
+		"pt"		"Zumbi Venenoso Fortifiacdo"
 	}
 	"Fortified Giant Poison Zombie"
 	{
 		"en" 		"Fortified Giant Poison Zombie"
 		"ru" 		"Крепкий гигантский ядовитый зомби"
+		"pt"		"Zumbi Venenoso Gigante Fortificado"
 	}
 	"Father Grigori"
 	{
 		"en" 		"Father Grigori"
 		"ru" 		"Отец Григорий"
+		"pt"		"Senhor Grigori"
 	}
 	"Metro Cop"
 	{
 		"en" 		"Metro Cop"
 		"ru" 		"Метрокоп"
+		"pt"		"Policial"
 	}
 	"Metro Raider"
 	{
 		"en" 		"Metro Raider"
 		"ru" 		"Метрорейдер"
+		"pt"		"Policial Invasor"
 	}
 	"Combine Rifler"
 	{
 		"en" 		"Combine Rifler"
 		"ru" 		"Комбайн с винтовкой"
+		"pt"		"Fuzileiro da Combine"
 	}
 	"Combine Shotgunner"
 	{
 		"en" 		"Combine Shotgunner"
 		"ru" 		"Комбайн с дробовиком"
+		"pt"		"Artilheiro da Combine"
 	}
 	"Combine Swordsman"
 	{
 		"en" 		"Combine Swordsman"
 		"ru" 		"Комбайн мечник"
+		"pt"		"Espadachim da Combine"
 	}
 	"Combine Elite"
 	{
 		"en" 		"Combine Elite"
 		"ru" 		"Элитный комбайн"
+		"pt"		"Elite da Combine"
 	}
 	"Combine Giant Swordsman"
 	{
 		"en" 		"Combine Giant Swordsman"
 		"ru" 		"Гигантский комбайн мечник"
+		"pt"		"Espadachim Gigante da Combine"
 	}
 	"Combine DDT"
 	{
 		"en" 		"Combine DDT"
 		"ru" 		"Комбайн DDT"
+		"pt"		"Combine DDT" // what the hell DDT stands for??
```

Comment:
what the hell DDT stands for??

Reply:
Its a short acronim for something, its better to keep it as how it is